where-to-travel/src/main/webapp/script.js

```diff
@@ -346,6 +346,12 @@ function populatePlaces(placeArray) {
         time: time,
       }
       ref.set(data);
+    } else if (firebase.auth().currentUser && $(this).hasClass('press') == false) {
+      const name = $(this).parent().parent().parent().attr('placeName');
+       // Delete user saved places when the star is not pressed
+      var uID = firebase.auth().currentUser.uid;
```

Comment:
Pull name, uID, and ref out of the if statements since it is shared in both branches?

Reply:
+1, name, uID and ref are defined on lines 336, 341, and 342 respectively already